tifeatures/dbmodel.py

```diff
@@ -74,10 +74,12 @@ def datetime_column(self, name: Optional[str] = None) -> Optional[Column]:
 
     def geometry_column(self, name: Optional[str] = None) -> Optional[GeometryColumn]:
         """Return the name of the first geometry column."""
-        if self.geometry_columns:
-            for col in self.geometry_columns:
-                if name is None or col.name == name:
-                    return col
+        if name and name.lower() == "none":
+            return None
```

Comment:
we treat `none`|`NONE` as a special case when user don't want geometry back

Reply:
I could argue that this logic as nothing to do in here and should be better implemented at `layer.Table` level with maybe a `no-geometry: bool` options